### PR TITLE
Basic Auth should be marked as deprecated for GCP

### PR DESCRIPTION
**Pull request by jlewi**
* See #1571
* Also reformat the existing instructions; break up the big code
  block of code setting environment variables into multiple steps.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/1630)

### CODE REVIEW

**Comment by jlewi** [comment]
Preview:
https://deploy-preview-1630--competent-brattain-de2d6d.netlify.com/docs/gke/deploy/deploy-cli/

File: content/docs/gke/deploy/deploy-cli.md [hunk 1]
```diff
@@ -62,52 +60,57 @@ some handy environment variables:
     gcloud auth application-default login
     ```
 
-1. Create environment variables to make the deployment process easier:
+1. Configure gcloud default default values for zone and project
```
Comment (richardsliu) [inline content/docs/gke/deploy/deploy-cli.md]: "default" is repeated

Comment (jlewi) [reply content/docs/gke/deploy/deploy-cli.md]: Thanks. PTAL

**Pull request merged**